android/books/src/main/java/kmp/android/books/navigation/BooksDestinations.kt

```diff
@@ -0,0 +1,12 @@
+package kmp.android.books.navigation
+
+import androidx.annotation.StringRes
+
+sealed class BooksDestinations(val route: String, @StringRes val titleRes: Int) {
+    object List : BooksDestinations("books/list", kmp.android.shared.R.string.books_view_toolbar_title)
```

Comment:
You can add import for `kmp.android.shared.R` and use only `R.string. ...` here and on the line 8.